pkg/azuredevops/smoke/submission.go

```diff
@@ -0,0 +1,511 @@
+package main
```

Comment:
I think it should be located in the github.tools.sap where submission pipeline is located.

It would be great to have possibility to change smoke tests in the same PR with changes of the submission pipeline.

Reply:
It should be discussed with the team. I wanted to make it available for other pipelines as well.

Reply:
ok, sure, it can be different repository than pipeline one but I think it should be github.tools.sap one as it is not used for OS releases and for smoke tests of ADO pipelines which has definitions in the github.tools.sap

@KacperMalachowski @dekiel @TorstenD-SAP @mm-mariani what do you think about it?